clients/python/client/osparc/_utils.py

```diff
@@ -90,3 +95,15 @@ async def _fcn_to_coro(callback: Callable[..., S], *args) -> S:
     """Get a coroutine from a callback."""
     result = await asyncio.get_event_loop().run_in_executor(None, callback, *args)
     return result
+
+
+def _sha256(file: Path) -> str:
```

Comment:
MINOR: the module `_utils` is protected but the function is to be used outside of this module. 
I suggest: `_utils.compute_sha256`

Reply:
Good point. I have adapted this.